### ISSUE TITLE
Add region tags

### ISSUE THREAD

**Issue by theantnest** [acceptbitcoincash/acceptbitcoincash#46]
Remembering that Bitcoin (Cash) is a global thing, it would be an amazing feature to add region tags for the merchants, and to allow the user to filter out things not available in their region.

**Comment by kenman345** [comment]
we do have a language tag for some of them. Would this maybe be useful?

**Comment by theantnest** [comment]
Not so much. Somebody that provides goods and services in the USA may not service Australia for example.

**Comment by kenman345** [comment]
this would be cool, I think we need to work on advanced filtering functionality soon anyways. Maybe if enough people agree, we can just script adding the region tag to anything thats not a .com site and then work from there?

**Comment by t0p0lin0** [comment]
Thinking about this more and more after witnessing multiple community submissions for many different places/merchants on Earth. I'm suggesting a new `key: value` pair in the YAML files, specified as:

`region: na` 

...with the region codes being (and listed in the [new issue](https://github.com/acceptbitcoincash/acceptbitcoincash/issues/new) placeholder):

Code | Name
-- | --
af | Africa
na | North America
oc | Oceania
an | Antarctica
as | Asia
eu | Europe
sa | South America

Then, we would need an additional UI filter for *Region*, offering choices of the continents above, and _Global_, which is basically clearing the region filter.

Thoughts? I'd like to have a clear path forward before finding time to tackle this task (it's game for anyone reading this mind you!).

**Comment by t0p0lin0** [comment]
Update: Proposed edits to YAML files:

`region: eu`
`country: es`
`city: Barcelona`

(_example of "The Dog is Hot" in **Food** category_)

**Comment by kenman345** [comment]
With the new request system coming, this is a feature set we can actually work on as more and more listings contain the tags. I think we should split this to two tasks on our project board. 

1. Adding in the region and other tags to our current listings.
2. Adding functionality to filter on these tags once step 1 is complete.

**Comment by t0p0lin0** [comment]
Certainly, and #1 is easy, but time-consuming. There's also issue #396 that will help if we wish to display a map. #2 is in progress in a `refactor/region-based` branch that I pushed a while ago, but it is far behind `master` and will need a fair amount of work to finalize.

**Comment by kenman345** [comment]
hey @jrkong can we use one of your scripts to identify the listings without these three tags? I know its most right now but if nothing else it gives us a way of keeping track of everything

**Comment by kenman345** [comment]
> Then, we would need an additional UI filter for Region, offering choices of the continents above, and Global, which is basically clearing the region filter.

@t0p0lin0 I just noticed that part. Shall I change the form to always include the region tag and assign "global" as the value? Seems odd as we use two letter values normally but if thats what we need for new additions to conform to what we want, then I am happy to have it in there.

**Comment by CreativeName4** [comment]
@GitCash give @kenman345 500 bits and an invite

**Comment by kenman345** [comment]
We've been fairly good about adding in the region tag where appropriate. I think this can be closed now and just maintain our same efforts moving forward. I am still hoping to offer advanced searching off this data in the future but this is just a lingering resolved issue now. 

Thanks for the suggestion and keep em coming if you see anything we can do better!
